cirrus/google_cloud/manager.py

```diff
@@ -334,29 +346,98 @@ def get_project_info(self):
 
         return response.json()
 
-    def get_buckets(self):
+    def get_bucket_iam_policy(self, bucket_name):
+        bucket = self._storage_client.get_bucket(bucket_name)
+        return bucket.get_iam_policy()
+
+    def create_bucket(
+            self, name, storage_class=None, public=False, requester_pays=False,
+            project=None, access_logs_bucket=None):
         """
-        Return all the buckets for the project
+        Create a Google Storage bucket.
 
         Returns:
-            List(google.cloud.storage.bucket.Bucket): Google Cloud Buckets
+            google.cloud.storage.bucket.Bucket: Google Cloud Bucket
+
+        Args:
+            name (str): Globally unique name for new Google Bucket
+            storage_class (str, optional): one of GOOGLE_STORAGE_CLASSES
+            public (bool, optional): Whether or not all data in bucket should
+                be open to the public (will only allow access by authN'd users
+                to support access logs)
+            requester_pays (bool, optional): Whether requester pays for API
+                requests for this bucket and its blobs.
+            project (str, optional): FIXME not currently used since Google's
+                Python API won't accept it... their docs are out of date and
```

Comment:
is this going to be supported soon? if not let's remove it?

Reply:
I read more docs, this is for passing a billing project to the bucket which we don't need to do during creation so I was mistaken. I've removed the param